js/pages/tablespage.ts

```diff
@@ -44,13 +44,15 @@ export class TablesPage extends PageBase implements ICurrentTableProvider {
     public nextGameTypeInformation: KnockoutComputed<string>;
     public splashShown = ko.observable(false);
     public tablesShown = ko.observable(true);
+    public progressBackgroundColor: KnockoutObservable<string>;
 
     constructor(private commandExecutor: ICommandExecutor) {
         super();
         const self = this;
         this.slideWidth = ko.observable(0);
         this.isConnectionSlow = ko.observable(false);
         this.calculateLandscapeWidth();
+        this.progressBackgroundColor = ko.observable($(".progress-background").css("background-color"));
```

Comment:
This value should be taken from configuration. No need to use jQuery.

Reply:
From what configuration? Do you mean that i should store variable `rgba(0,0,0,.75)` in appConfig ?

Reply:
Yes, something like that